CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
+## 0.8.4
+* Adding support for `tile.type` using the "class" attribute for Tiled 1.9's Unified Custom Types
+
 ## 0.8.3
 * Downgrade meta dependency
 
-## 0.8.2
-* Add support for class, which is replacing type in tiled 1.9
```

Comment:
You should never remove entries from the changelog, if the entry was wrong it should be explained in the new entry.